Swiftfin tvOS/Views/ItemView/SeriesItemView/Components/SeriesEpisodesView.swift

```diff
@@ -99,8 +106,6 @@ extension SeriesEpisodesView {
         private var lastFocusedEpisodeID: String?
         @State
         private var currentEpisodes: [BaseItemDto] = []
-        @State
-        private var wrappedScrollView: UIScrollView?
```

Comment:
I tried to fix those things that were annoying me the most ;) I'm new to Jellyfin/Swiftfin, but since I've done a very similar project a few years ago in ObjC for Kodi RPC I wanted to see if I can contribute somehow :) Anyway, thanks for taking care of it in your rewrite PR, hope to see those changes soon in the official app :)

Reply:
btw. i've checked out your branch and this issue is not fixed there yet. It would be great if you could add a fix for the season buttons not scrolling to the focus point.

Reply:
https://user-images.githubusercontent.com/2404568/226182315-095bce41-db6a-4ec7-99da-d5e0d29a6162.mov

Reply:
@LePips ^^

Reply:
maybe file a new bug @shadzik ?